src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -202,4 +204,25 @@ export class ServicesComponent extends BaseComponent<IServicesComponentPropertie
 
         return ListComponent.renderColumn(textToRender || "", ListComponent.defaultColumnRenderer, colDataClassName);
     }
+
+    private static _getServiceStatus(service: IServiceItem, cssClassName: string): React.ReactNode {
+        let statusProps: IStatusProps = Statuses.Success;
+        let tooltipText: string | undefined;
+        if (service.type == "LoadBalancer") {

```

Comment:
"LoadBalancer" --> take this as enum, like a const etc

also check the condition like
if (service.type === SomeConst) {

Reply:
Done. Made `LoadBalancer` a const and changed the comparision.